src/main/java/core/basesyntax/service/ReportWriter.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class ReportWriter {
+    public static void writeReport(
+            Map<String, Integer> inventory, String outputFile) throws IOException {
```

Comment:
```suggestion
    public static void writeReport(String outputFile) throws IOException {
```
Add Storage class with static map that imitates real DB